src/Akeneo/Platform/Bundle/ImportExportBundle/Resources/translations/jsmessages.en_US.yml

```
@@ -164,6 +164,7 @@ pim_import_export:
                 compute_family_variant_structure_changes: Compute family variant structure changes
                 compute_completeness_of_products_family: Compute completeness of products family
                 mass_delete: Mass delete products
+                clean_removed_attribute_job: Clean removed attribute values in products & product models
```

As in EE we will clean proposal values, we should be less specific
```suggestion
                clean_removed_attribute_job: Clean removed attribute values
```